src/verifai/server.py

```diff
@@ -144,6 +144,8 @@ def __init__(self, sampling_data, monitor, options={}):
                 sampler_params=params
             )
 
+        if (self.sample_space.dynamicFeatureNamed) != 0:
```

Comment:
Rather than peeking inside like this, how about defining a flag like `FeatureSpace.hasTimeSeries` and using it here and in a couple other places where you do a check like this?